Backpack/src/main/java/net/skyscanner/backpack/snackbar/BpkSnackbar.kt

```diff
@@ -70,6 +70,7 @@ class BpkSnackbar private constructor(
 
   private val snackbar = Snackbar.make(view, "", duration).apply {
     setBackgroundColorCompat(backgroundColor)
+    animationMode = BaseTransientBottomBar.ANIMATION_MODE_SLIDE
```

Comment:
Does this behave any different for the user? Have we disabled animations when running tests?